From: Julian Ospald <julian.ospald@googlemail.com>
Date: Fri Apr 13 18:44:35 UTC 2012

fix "QA Notice: The following shared libraries lack a SONAME"
respect LDFLAGS and CFLAGS

--- Makefile
+++ Makefile
@@ -44,7 +44,7 @@
 	$(CC) $(CPPFLAGS_FAST) $(CFLAGS_LIB) -c http_parser.c -o libhttp_parser.o
 
 library: libhttp_parser.o
-	$(CC) -shared -o libhttp_parser.so libhttp_parser.o
+	$(CC) $(CFLAGS) $(LDFLAGS) -Wl,-soname,libhttp_parser.so -shared -o libhttp_parser.so libhttp_parser.o
 
 package: http_parser.o
 	$(AR) rcs libhttp_parser.a http_parser.o
